pkg/nas/internal/config.go

```diff
@@ -126,20 +126,19 @@ func GetNodeConfig() (*NodeConfig, error) {
 		klog.Info("enabled nas volume stats")
 	}
 
-	// get node info
-	nodeName := os.Getenv("KUBE_NODE_NAME")
-	node, err := kubeClient.CoreV1().Nodes().Get(context.Background(), nodeName, metav1.GetOptions{})
-	if err != nil {
-		return nil, err
-	}
-	config.NodeName = nodeName
-
 	// check if losetup enabled
 	if value := os.Getenv("NAS_LOSETUP_ENABLE"); value != "" {
 		config.EnableLosetup, _ = parseBool(value)
 	}
 	if config.EnableLosetup {
 		klog.Info("enabled nas losetup mode")
+		nodeName := os.Getenv("KUBE_NODE_NAME")
+		node, err := kubeClient.CoreV1().Nodes().Get(context.Background(), nodeName, metav1.GetOptions{})
+		if err != nil {
+			return nil, err
+		}
+		config.NodeName = nodeName
```

Comment:
`config.NodeName` is also used as CSI NodeID, So I think we still need this when `!config.EnableLosetup`

Reply:
CI failure also caused by this issue:

```
  I1226 18:57:46.163485   18655 main.go:109] "Received NotifyRegistrationStatus call" status="&RegistrationStatus{PluginRegistered:false,Error:RegisterPlugin error -- plugin registration failed with err: error adding CSI driver node info: driverNodeID must not be empty,}"
  E1226 18:57:46.163520   18655 main.go:111] "Registration process failed with error, restarting registration container" err="RegisterPlugin error -- plugin registration failed with err: error adding CSI driver node info: driverNodeID must not be empty"
```

Reply:
> How about renaming sidecar mode's nodeName to podName

I think we can still use KUBE_NODE_NAME env name, and let caller to determine what value is appropriate.